### PR TITLE
Switch from Travis to Github Actions

### LINKED ISSUE #1128: Add Ruby 2.0 testing to PRs
See https://github.com/theforeman/forklift/pull/1127

### PR DESCRIPTION
**Pull request by ekohl**
Fixes #1128

### CODE REVIEW

**Comment by ekohl** [comment]
Ansible lint is broken on the `to_nice_yaml` represenation (https://github.com/ansible/ansible-lint/pull/51) and it's finding all kinds of lint violations. I'm wondering if we ever actually ran it. Also not sure why Travis doesn't find the 2.0.0 RVM. Perhaps that's simply not available anymore.

**Comment by ehelms** [comment]
Looks like the Travis changes are breaking things... I'd be good with leaving the `.travis.yml` alone and having both CI systems running in parallel during the conversion. This should require less having to fiddle with Travis and faster GH spinup.

**Comment by ekohl** [comment]
I've split off https://github.com/theforeman/forklift/pull/1166 & https://github.com/theforeman/forklift/pull/1167 so they at least can get merged.

File: playbooks/runcible.yml [hunk 1]
```diff
@@ -36,10 +36,10 @@
       become: true
 
     - name: "Install RVM gpg key"
-      shell: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
+      command: "gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3"
 
     - name: "Install RVM"
-      shell: "curl -sSL https://get.rvm.io | bash -s stable --ruby"
+      command: "curl -sSL https://get.rvm.io | bash -s stable --ruby"
```
Comment (evgeni) [inline playbooks/runcible.yml:42]: a pipe can't be in a `command`

**Comment by evgeni** [comment]
if you push an empty `.github/workflows/ansible.yml` to `master`, this PR would trigger GH actions, otherwise it does not :(

**Comment by ekohl** [comment]
Since it ran in mine, I can tell you that ansible-lint just hard fails: https://github.com/ekohl/forklift/runs/734487554?check_suite_focus=true. I don't want to spend time on Ansible so closing this.

**Pull request closed without merging**